### PR TITLE
Checked for typos in comments.

### PR DESCRIPTION
**Pull request by kandelvijaya**

### CODE REVIEW

**Comment by zhigang1992** [comment]
Thanks for doing this,

Can you help update this PR and let's ship it.

🍻

File: Source/SwiftyJSON.swift [hunk 1]
```diff
@@ -58,14 +58,14 @@ public struct JSON {
...
+     - parameter data:  The NSData used to convert to json. Top level object in data is an NSArray or NSDictionary
      - parameter opt:   The JSON serialization reading options. `.AllowFragments` by default.
-     - parameter error: error The NSErrorPointer used to return the error. `nil` by default.
-     
+     - parameter error: The NSErrorPointer used to return the error. `nil` by default.
+
      - returns: The created JSON
      */
-    public init(data:Data, options opt: JSONSerialization.ReadingOptions = .allowFragments, error: NSErrorPointer = nil) {
+    public init(data:NSData, options opt: NSJSONReadingOptions = .AllowFragments, error: NSErrorPointer = nil) {
```
Comment (zhigang1992) [inline Source/SwiftyJSON.swift]: We are currently supporting swift 3 and Xcode 8 on master, this change reverted part of that effort. :smile:

File: Source/SwiftyJSON.swift [hunk 2]
```diff
@@ -78,14 +78,14 @@ public struct JSON {
...
      - parameter string: Normal json string like '{"a":"b"}'
-     
+
      - returns: The created JSON
      */
-    public static func parse(_ string:String) -> JSON {
-        return string.data(using: String.Encoding.utf8)
-            .flatMap({JSON(data: $0)}) ?? JSON(NSNull())
+    public static func parse(string:String) -> JSON {
+        return string.dataUsingEncoding(NSUTF8StringEncoding)
```
Comment (zhigang1992) [inline Source/SwiftyJSON.swift]: Here as well.

**Comment by kandelvijaya** [comment]
I will fix and create a new PR

**Pull request closed without merging**